ruby-3.0.yaml

```diff
@@ -58,8 +58,6 @@ pipeline:
   - uses: autoconf/make
   - uses: autoconf/make-install
   - uses: strip
-
-  # Remove gem bundler installed with ruby. It's provided by it's own package
```

Comment:
(This is a bug in https://github.com/go-yaml/yaml that I want to fix for our usage)

Reply:
Oh whoops, should I try to put the comment back>? What's the easiest way to keep it without messing up automation?

Reply:
I don't think we have a great answer.

If it's not too hard, it'd be nice to keep the comment, but if you run any of our YAML tools (wolfictl or melange) it'll happen again — they all use that YAML library.

I'm going to try to get this fixed in a fork in the next few days.